tests/conftest.py

```diff
@@ -350,6 +364,9 @@ def stop_fpmsyncd(dvs):
     def init_asicdb_validator(self):
         self.asicdb = AsicDbValidator(self)
 
+    def flush_appdb(self):
```

Comment:
Where is this used?   If we need flush appDB, I would think we need flush some other DBs too, like stateDB etc.

Reply:
Nowhere. Kept it as primitive method